css-canvas-medical/index.html

```diff
@@ -0,0 +1,100 @@
+        <button class="btn">Implementation</button>
+        <button class="btn">About Us</button>
+        <button class="btn">Blog</button>
+        <button class="btn" id="blue">Contact Us</button>
+      </div>
+    </div>
+    <div class="row">
+      <div class="overlay"></div>
+      <div class="img-text">
+        <h1 class="img-head">The EMR Reimagined for Primary Care</h1>
+        <p class="text">Can't remember the last time you talked with a patient without  focusing
+          on the EMR? Canvas lets you rediscover joy in the work you do. Capture meaningful data, identify gaps in care,
+          seamlessly between tasks within the platform. You stay present with patients
+          while your whole care team stays up-to-speed. Be a better doctor with Canvas, designed for primary care providers with the
+          help of primary care providers.
+        </p>
+        <button class="start-button">Get Started</button>
+      </div>
+      <img src="./images/hero.jpg" alt="hero" id="hero">
+    </div>
+    <div class="row-mid">
+      <div class="column-mid">
+        <h1 class="mid-header">Clinitians Prefer Canvas</h1>
+        <p class="small-font">Physician-owned primary care practices across the country have already
+         made Canvas the fabric of their care
+        </p>
+      </div>
+    </div>
+    <div class="row-addresses">
+      <div class="column-addresses">
+        <p class="addresses">Cascade Family Practice, Portland OR</p>
+        <p class="addreses">Temescal Creek Medicine, Oakland CA</p>
+        <p class="addreses">San Juan Healthcare, Friday Harbor WA</p>
+      </div>
+      <div class="column-addresses">
+        <p class="addresses">Ward Medical Clinic, Bonifay FL</p>
+        <p class="addreses">Caribou Medical, Towanda PA</p>
+        <p class="addreses">Bay Tree Family Care, Red Bay AL</p>
+      </div>
+    </div>
+    <div class="row-ready">
+      <div class="column-half">
+        <h1 class="ready-header">Ready to learn more?</h1>
+        <p class="ready-para">Schedule a private meeting now</p>
+      </div>
+      <div class="column-half">
+        <div class="start">
+          <button class="start-button">Get Started</button>
+        </div>
+      </div>
+    </div>
+    <div class="black-box">
+      <div class="column-bottom-one">
+        <img src="./images/logo-white.png" id="logo">
+        <p id="logo-small">2307 Irving Street, Suite 228 <br>
+          San Francisco CA 94122<br>
+          (800)370-1416<br>
+          info@canvasmedical.com<br>
+        </p>
+      </div>
+      <div class="column-quarter font-bottom">
+        <p>COMPANY</p>
+        <p>About us</p>
+        <p>Blog</p>
+      </div>
+      <div class="column-quarter font-bottom">
+        <p>PRODUCT</p>
+        <p>Product</p>
+        <p>Implementation</p>
+        <p>For Health Plans</p>
+        <p>Security</p>
+      </div>
+    </div>
+    <div class="row-footer">
+      <div class="column-footer">
```

Comment:
You shouldn't have `column` classes named after what they are being applied to, they should be named after their size and only have `flex-basis` or `width` applied to them. Just like with the `row` classes I already mentioned, any additional styling should be applied to other classes you can add to the element in addition to the `column` class for setting its width.